server/cluster_worker.go

```diff
@@ -679,16 +338,16 @@ func (c *raftCluster) callCommand(request *raft_cmdpb.RaftCmdRequest) (*raft_cmd
 	return msg.CmdResp, nil
 }
 
-func (c *raftCluster) getRegionLeader(regionID uint64, peer *metapb.Peer) (*metapb.Peer, error) {
+func (c *raftCluster) getRegionDetail(regionID uint64, peer *metapb.Peer) (*raft_cmdpb.RegionDetailResponse, error) {
```

Comment:
Not need, but i am not sure whether we need it in the further.
What is your opinion? Remove it now?